spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerPersistentEntityImpl.java

```diff
@@ -157,6 +159,9 @@ public void addPersistentProperty(SpannerPersistentProperty property) {
 		else if (!property.isInterleaved()) {
 			this.columnNames.add(property.getColumnName());
 		}
+		else if (property.isInterleaved() && !property.isLazyInterleaved()) {
+			this.hasEagerlyLoadedProperties = true;
+		}
```

Comment:
Leave a comment about what should happen if the property is interleaved lazily (that it's skipped and not persistent)?